README.md

```diff
@@ -33,6 +33,11 @@ The AI Agent Contract Tools template is a **MINIMAL** template to build an AI Ag
 - ⛑️ Secure: Execution guarded by rock solid TEE / Intel SGX
 - 🔒 Private: Host API keys and user privacy at ease
 - 💎 Unstoppable: Powered by IPFS and Phala's 35k+ decentralized TEE workers
+- :fire: [**hono/tiny** Support](https://hono.dev/docs/api/presets#hono-tiny): Currently support a limited implementation of Hono.
```

Comment:
Do we support the URL paths? I did not have time to test, but can we support full hono today? I was adding this to ensure devs don't try full  hono version if we don't support it out-of-box

Reply:
The main difference between the full version and the tiny version is in their routing path parsing algorithm and the final bundle size.

> It's ideal for long-life servers built with Deno, Bun, or Node.js. For environments such as Cloudflare Workers, Deno Deploy, where v8 isolates are utilized, this preset is suitable as well. Because the isolations persist for a certain amount of time after booting.

I propose using `@hono/tiny` to avoid a large bundle size and the 20MB final artifact limitation.

Reply:
Using `@hono/tiny` can reduce the file size by 20kb.

```
❯ bat src/index.ts | head 
import '@phala/wapo-env'
import { Hono } from 'hono/tiny'
import { handle } from '@phala/wapo-env/guest'

export const app = new Hono()

app.get('/', (c) => {
  const derivedKey = Wapo.deriveSecret("SALTY");
  const hex = Buffer.from(derivedKey).toString('hex');
  return c.json({ message: `${hex}` })
  
❯ npm run build

> minimal-wapo-ts-starter@1.0.0 build
> tsup --config tsup.config.ts

CLI Building entry: src/index.ts
CLI Using tsconfig: tsconfig.json
CLI tsup v8.2.4
CLI Using tsup config: /home/leechael/workshop/phala/minimal-wapo-ts-starter/tsup.config.ts
CLI Target: es2020
CLI Cleaning output folder
IIFE Build start
IIFE dist/index.js     108.14 KB
IIFE dist/index.js.map 144.27 KB
IIFE ⚡️ Build success in 17ms


❯ bat src/index.ts| head 
import '@phala/wapo-env'
import { Hono } from 'hono'
import { handle } from '@phala/wapo-env/guest'

export const app = new Hono()

app.get('/', (c) => {
  const derivedKey = Wapo.deriveSecret("SALTY");
  const hex = Buffer.from(derivedKey).toString('hex');
  return c.json({ message: `${hex}` })
  
❯ npm run build          

> minimal-wapo-ts-starter@1.0.0 build
> tsup --config tsup.config.ts

CLI Building entry: src/index.ts
CLI Using tsconfig: tsconfig.json
CLI tsup v8.2.4
CLI Using tsup config: /home/leechael/workshop/phala/minimal-wapo-ts-starter/tsup.config.ts
CLI Target: es2020
CLI Cleaning output folder
IIFE Build start
IIFE dist/index.js     128.98 KB
IIFE dist/index.js.map 177.71 KB
IIFE ⚡️ Build success in 15ms
```

Reply:
Done